@navikt/core/react/src/copybutton/CopyButton.tsx

```diff
@@ -63,6 +63,11 @@ export interface CopyButtonProps
    * @default 'Kopiert'
    */
   activeTitle?: string;
+  /**
+   * Icon position in Button
```

Comment:
Liten "b" kanskje? Siden det ikke er snakk om Button-komponenten.
```suggestion
   * Icon position in button
```